vip-cache-manager/vip-cache-manager.php

```diff
@@ -61,6 +64,79 @@ public function clear_queued_purge_urls() {
 		$this->purge_urls = [];
 	}
 
+	/**
+	 * Display a button to purge the cache for the specific URL and its assets
+	 *
+	 * @return void
+	 */
+	public function admin_bar_callback( WP_Admin_Bar $admin_bar ) {
+		if ( is_admin() || ! current_user_can( 'manage_options' ) ) {
+			return;
+		}
+
+		$admin_bar->add_menu(
+			[
+				'id'     => 'vip-purge-page',
+				'parent' => null,
+				'group'  => null,
+				'title'  => 'Flush Cache for Page',
+				'href'   => '#',
```

Comment:
Could the `meta` -> `html` value be set, so that this becomes a `<button type="button">` rather than an empty `<a>` link (well, link pointing to `#`), for accessibility purposes?

Reply:
@GaryJones unfortunately, I wasn't able to find an example where it works out of the box, setting `html` property does allow to add _extra_ HTML, but it doesn't replace the node.

Reply:
Oh, yes it [adds extra html](https://github.com/WordPress/WordPress/blob/d7c0343d5f7e0efdad5b7dce64981e74f5d0b7a0/wp-includes/class-wp-admin-bar.php#L575-L577). TIL. Thanks for looking!